guides/common/modules/proc_acd_installation.adoc

```diff
@@ -45,6 +45,44 @@ endif::[]
 # {foreman-maintain} service restart
 ----
 
+[id="{context}_installation_{SmartProxyServer}"]
+= Installing ACD on {SmartProxyServer}
+
+You need to install two packages to use ACD on your {SmartProxy}.
+
+.Prerequisites
+* {ProjectServer} with ACD xref:{context}_installation[installed].
+
+.Procedure
+ifdef::foreman-el,foreman-deb,katello,satellite[]
+. Create a yum repository `/etc/yum.repos.d/foreman-plugins.repo` on your {SmartProxyServer}:
++
+[source,bash,subs="+quotes,attributes"]
+----
+[foreman-plugins]
+name=Foreman plugins
+baseurl=https://yum.theforeman.org/plugins/nightly/el7/x86_64/
+enabled=1
+gpgcheck=0
+----
+endif::[]
+. Install the required packages on your {SmartProxyServer}:
++
+[source,bash]
+----
+# yum install tfm-rubygem-smart_proxy_acd tfm-rubygem-smart_proxy_acd_core
+----
+. Restart the {SmartProxyServer} services:
++
+[source,bash]
+----
+# systemctl restart foreman-proxy smart_proxy_dynflow_core
+----
+. Refresh {SmartProxy} features on {ProjectServer}:
```

Comment:
Please work with the installer team to get it integrated in the installer. The `tfm-` instructions are incorrect on EL8 which doesn't have SCLs. `smart_proxy_dynflow_core` is already gone in nightly so this is incorrect. Refreshing of features (including checks that Foreman recognizes the feature) is also part of it. The installer takes care of all of that.

Reply:
As of right now, I only know how it works on EL7 with Foreman 2.3. 

I will get back to you on how I want to procede soon.

Reply:
Update: We have plans to integrate ACD into the Foreman installer in the future.

Would you be happy with either a bigger disclaimer (regarding EL7 and Foreman 2.3) or only merging it into the 2.3 branch?

Else, I can only wait until we add it to the Foreman installer and/or support Foreman ACD on a stable Foreman version without `smart_proxy_dynflow_core`, which [seems to be 2.5](https://community.theforeman.org/t/phasing-out-smart-proxy-dynflow-core/22841/8?u=maximilian).

Reply:
> Update: We have plans to integrate ACD into the Foreman installer in the future.

The patches have been merged to puppet-foreman and puppet-foreman_proxy but there is still a patch missing in foreman-installer. That should really be done.



> Would you be happy with either a bigger disclaimer (regarding EL7 and Foreman 2.3) or only merging it into the 2.3 branch?

I think it's ok to merge a slightly different version for 2.3 but remember that Foreman 2.3 is EOL.

Reply:
@ekohl if you say there need to be a change in foreman-installer, you mean in https://github.com/theforeman/foreman-installer/blob/eb73fb7f4b2fe6470816a8707bae433e2b2cd71d/config/foreman-answers.yaml ?

Reply:
That and also katello-answers.yaml. If there's a smart proxy component (which there is) then also foreman-proxy-content. Don't forget to add a migration. There's bin/create-migration for that (https://github.com/theforeman/foreman-installer/pull/699 may be good to use as a bugfix).